packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -60,6 +60,27 @@ abstract class RendererBinding extends BindingBase implements SchedulerBinding,
           return _forceRepaint();
         }
       );
+      registerBoolServiceExtension(
+          name: 'debugPaintBaselinesEnabled',
+          getter: () async => debugPaintBaselinesEnabled,
+          setter: (bool value) async {
```

Comment:
Sure - I updated `debugPaintBaselinesEnabled` to be like the `debugPaintSizeEnabled` implementation.